### ISSUE TITLE
Year jumps unexpectedly after guitar pick drag

### ISSUE THREAD

**Issue by minniewong** [nasa-gibs/worldview#761]
Steps to reproduce: 
Start app, click on down arrow for the Year, the timeline does not update/change/move to correct year and the imagery does not update either.

**Comment by Benjaki2** [comment]
Issue is ready for review but depends on PR #778. This work will be rebased on `develop` once sub-daily timeline work gets merged into `develop`.
